CONTENT/about.markdown

```diff
@@ -1,6 +1,6 @@
 [//]: # (@TODO - make this file nice)
 
-# Manifesto
+# About Wizard Amigos Institute
 (Philosophy/Values/Goals)
 => Link to Github
```

Comment:
What about just `About`?
If you think `About Wizard Amigos Institute` ist better, I'm also fine with that...